platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -1601,7 +1601,10 @@ public void invalidateCustomLayers() {
         mMapView.invalidateCustomLayers();
     }
 
-    MapView getMapView() {
+    /**
+     * Do not use this method, used by Mapbox internally
+     */
+    public MapView getMapView() {
```

Comment:
@ivovandongen This seems unnecessary for this PR, I'm making the visibility private again. Is it needed somewhere else?